### ISSUE TITLE
Request: add callback when ZIP is done writing

### ISSUE THREAD

**Issue by laurenceks** [7th-Cyborg/vite-plugin-zip-pack#6]
To save running zipPack twice, I have written a small custom plugin to copy the ZIP into another directory (for automatically uploading to Google Drive).

Currently `zipPack `reports "done" before the file is actually done writing. It looks like this is because `createZipArchive` uses `.generateAsync`, which is obviously is async. So when the next plugin looks for the file using` "./build/*.zip"` it can't find it. As a workaround I'm using a watcher for waiting for the file to be added and then copying it.

```
const watcher = chokidar.watch("./build/*.zip", {
                    awaitWriteFinish: true,
                });

                watcher.on("add", () => {
                     //ZIP has finished writing and now I can copy it
                 });
```

It would be helpful to have a callback or some kind of `awaitFinish` flag for allowing `createZipArchive` to finish before moving on to the next plugin.

**Comment by reinerBa** [comment]
Cool and thanks for the lib. Before that i had an extra file for zipping the output. Do you need a CI-Step that publishes the package to npm?

**Comment by 7th-Cyborg** [comment]
No problem, thanks for contributing to lib, glad to see you find it useful. There is currently no need for npm CI-Step but thanks for asking

**Comment by reinerBa** [comment]
Your welcome. If you decide to use npm CI-Step, you can use this YML-File i wrote last week for this job. 
https://github.com/reinerBa/vite-plugin-stopdev/blob/primary/.github/workflows/release-tag.yml 
It works like a charme 🤞

**Comment by 7th-Cyborg** [comment]
Ok, thanks

**Comment by 7th-Cyborg** [comment]
Included in release v1.1.0
